cdap-docs/developers-manual/source/building-blocks/datasets/table.rst

```diff
@@ -207,3 +207,16 @@ When you want to delete all the columns of a row and you know all of them,
 passing all of them will make the deletion faster. Deleting all the columns of a row will
 also delete the entire row, as the underlying implementation of a Table is a 
 `columnar store. <http://en.wikipedia.org/wiki/Column-oriented_DBMS>`__
+
+.. _table-datasets-pre-splitting:
+
+Pre-Splitting a Table into Multiple Regions
+===========================================
+
+To create a Dataset pre-split into multiple regions, where the underlying storage is HBase,
+you can use::
```

Comment:
Is this line of code correct? ``DatasetProperties.builder().add("hbase.splits", "[[64],[128],[192]]").build();``
Because I got the impression from the conversation that the prop wanted some like this:
````
DatasetProperties.builder().add("hbase.splits", new Gson().toJson(new byte[][]{{64},{128},{192}})).build();
```